tools/plugins/com.liferay.ide.core/src/com/liferay/ide/core/util/FileUtil.java

```diff
@@ -391,7 +391,7 @@ public static String readContents( InputStream contents ) throws IOException
         return out.toString();
     }
 
-    public static String[] readLinesFromFile( File file )
+    public static String[] readLinesFromFile( File file, boolean includeNewlines )
```

Comment:
When you add a new version of a method, always add a wrapper around the old way you called it to keep from having to break those callers.